source/utilities.tex

```diff
@@ -6764,33 +6764,43 @@
         I uninitialized_fill_n(I first, iter_difference_t<I> n, const T& x);
   }
 
+  // \ref{specialized.construct}, \tcode{construct_at}
+  template<class T, class... Args>
+    constexpr T* construct_at(T* location, Args&&... args);
+
+  namespace ranges {
+    template<class T, class... Args>
+      constexpr T* construct_at(T* location, Args&&... args);
+  }
+
+  // \ref{specialized.destroy}, \tcode{destroy}
   template<class T>
-    void destroy_at(T* location);
+    constexpr void destroy_at(T* location);
   template<class ForwardIterator>
-    void destroy(ForwardIterator first, ForwardIterator last);
+    constexpr void destroy(ForwardIterator first, ForwardIterator last);
   template<class ExecutionPolicy, class ForwardIterator>
-    void destroy(ExecutionPolicy&& exec, // see \ref{algorithms.parallel.overloads}
+    constexpr void destroy(ExecutionPolicy&& exec,              // see \ref{algorithms.parallel.overloads}
                  ForwardIterator first, ForwardIterator last);
```

Comment:
The parameters need to be indented to match up with the (, same for other modified functions that wrap lines - fix them all.